Sources/Basic/TemporaryFile.swift

```diff
@@ -74,6 +74,9 @@ public final class TemporaryFile {
 
     /// FileHandle of the temporary file, can be used to read/write data.
     public let fileHandle: FileHandle
+    
+    /// Whether the file should be deleted on dealloc.
+    public let deleteOnClose: Bool
```

Comment:
Can you propose this as its own PR?

Reply:
https://github.com/apple/swift-package-manager/pull/1625